START.htm

```diff
@@ -177,17 +177,17 @@ <h2 id="update">Update board with latest software</h2>
     <img src="Docs/images/extract-latestimage.png" width="50%" /><br />
 
     <h3>Step #4: Install SD card programming utility</h3>
-    <p>Download and install <a target="_blank" href="https://launchpad.net/win32-image-writer/+download">Image Writer for Windows</a>. Be sure to download the binary distribution.</p>
-    <img src="Docs/images/download-win32diskimager.png" width="50%" /><br />
-    <img src="Docs/images/extract-win32diskimager.png" width="50%" /><br />
+    <p>Download and install <a target="_blank" href="https://etcher.io/">Etcher for Windows</a>.</p>
+    <img src="Docs/images/download-etcher.png" width="50%" /><br />
+    <img src="Docs/images/install-etcher.png" width="50%" /><br />
     <p>Some general help on programming SD cards can be found on the <a target="_blank" href="http://help.ubuntu.com/community/Installation/FromImgFiles">Ubuntu Image Writer page</a>. 
 
     <h3>Step #5: Connect SD card to your computer</h3>
     <p>Use the provided microSD card to SD adapter or a USB adapter to connect the SD card to your computer.</p>
 
     <h3>Step #6: Write the image to your SD card</h3>
-    <p>Use either the Ubuntu Image Writer or instructions on its page to write the decompressed image to your SD card.</p>
-    <img src="Docs/images/write-latestimage.png" />
+    <p>Use Etcher to write the decompressed image to your SD card.</p>
```

Comment:
https://github.com/jadonk/beaglebone-getting-started/pull/30/files#diff-3e3d57b22232bb70d800b4f6bf126024R167 says "The file you download will have an .img.xz extention." ?